config/src/apps/vespa-get-config/getconfig.cpp

```diff
@@ -71,23 +72,22 @@ GetConfig::usage()
 void
 GetConfig::initRPC(const char *spec)
 {
-    _supervisor = new FRT_Supervisor();
+    _server = std::make_unique<fnet::frt::StandaloneFRT>();
+    _supervisor = &_server->supervisor();
     _target     = _supervisor->GetTarget(spec);
-    _supervisor->Start();
 }
 
 
 void
 GetConfig::finiRPC()
 {
-    if (_target != NULL) {
+    if (_target != nullptr) {
         _target->SubRef();
-        _target = NULL;
+        _target = nullptr;
     }
-    if (_supervisor != NULL) {
-        _supervisor->ShutDown(true);
-        delete _supervisor;
-        _supervisor = NULL;
+    if (_server) {
+        _server.reset();
+        _supervisor = nullptr;
```

Comment:
prefer setting supervisor to nullptr first

Reply:
Fixed